Cargo.toml

```diff
@@ -42,7 +42,7 @@ repository.workspace = true
 readme.workspace = true
 documentation = "https://docs.rs/libcrux/"
 description = "The Formally Verified Cryptography Library"
-exclude = ["/tests", "/specs"]
```

Comment:
Is there a need to exclude other things like the `wasm-demo`, `fuzz` and `formal_verification` directories, or things like `add-c-kyber-to-nss.py`? Or is that not included for some reason by default?